src/Adapters/Auth/index.js

```diff
@@ -41,18 +45,22 @@ const providers = {
   vkontakte,
   qq,
   wechat,
-  weibo
-}
+  weibo,
+  xiaomi
+};
 
 function authDataValidator(adapter, appIds, options) {
-  return function(authData) {
-    return adapter.validateAuthData(authData, options).then(() => {
-      if (appIds) {
+  return function (authData)
+  {
+    return adapter.validateAuthData(authData, options).then(() =>
+    {
+      if (appIds)
+      {
         return adapter.validateAppId(appIds, authData, options);
       }
       return Promise.resolve();
     });
-  }
+  };
```

Comment:
Can you revert these style changes? 

Generally you should avoid unnecessary stylistic changes when adding a feature unless it's part of the change itself (i.e. specifically a stylistic change PR). In this case it's superfluous and adds to what we have to review. It'll make it _much_ easier for us to go through and review your changes if they're only functional changes.